### PR TITLE
A little of refactor in task manager and task keeper

### PR DESCRIPTION
**Pull request by marmistrz**

### CODE REVIEW

File: golem/task/taskmanager.py [hunk 1]
```diff
@@ -13,20 +13,21 @@
...
+    ResourceType
 from golem.task.taskkeeper import CompTaskKeeper, compute_subtask_value
 from golem.task.taskstate import TaskState, TaskStatus, SubtaskStatus, \
     SubtaskState
 
 logger = logging.getLogger(__name__)
 
 
-def log_subtask_key_error(*args, **kwargs):
+def log_subtask_key_error(*args):
```
Comment (jiivan) [inline golem/task/taskmanager.py:25]: Unless you change `golem.core.common.HandleError.__call__` decorator it won't work. What is the reason of this change?

File: golem/task/taskmanager.py [hunk 2]
```diff
@@ -13,20 +13,21 @@
...
     logger.warning("This is not my subtask {}".format(args[1]))
     return None
 
 
-def log_task_key_error(*args, **kwargs):
+def log_task_key_error(*args):
```
Comment (jiivan) [inline golem/task/taskmanager.py:30]: Unless you change `golem.core.common.HandleError.__call__` decorator it won't work. What is the reason of this change?

**Review by jiivan — CHANGES_REQUESTED** [review]

File: golem/task/taskmanager.py (hunk 2 above)
Comment (marmistrz) [reply golem/task/taskmanager.py:30]: This was fixed while merging @Wolf480pl's change: https://github.com/golemfactory/golem/commit/56a7944f9305666dfdada0d3870ea394b1668d5e

File: golem/task/taskmanager.py [hunk 3]
```diff
@@ -178,7 +178,7 @@ def dump_task(self, task_id):
             logger.debug('DUMP TASK %r', filepath)
             with filepath.open('wb') as f:
                 pickle.dump(data, f, protocol=2)
-        except:
+        except Exception:
```
Comment (maaktweluit) [inline golem/task/taskmanager.py:181]: Maybe catch a more specific exception if possible?

**Review by maaktweluit — COMMENTED** [review]
LGTM!

When FIXME is fixed ( or made as issue ) will approve

**Comment by jiivan** [comment]
Closed in favor of #1581

**Pull request closed without merging**